src/tribler/core/upgrade/tests/test_upgrader.py

```diff
@@ -194,6 +196,41 @@ def _exists(db, table, column):
         assert mds.get_value('db_version') == '14'
 
 
+def test_upgrade_pony14to15(upgrader: TriblerUpgrader, channels_dir, trustchain_keypair, mds_path):
+    _copy(source_name='pony_v14.db', target=mds_path)
+
+    now = int(time.time())
+
+    mds = MetadataStore(mds_path, channels_dir, trustchain_keypair, check_tables=False)
+    with db_session:
+        mds.TorrentState(infohash=unhexlify(b'0001'), seeders=0, leechers=0, self_checked=0, has_data=0, last_check=0)
+        mds.TorrentState(infohash=unhexlify(b'0002'), seeders=1, leechers=2, self_checked=0, has_data=1,
+                         last_check=now - 1000)
+        mds.TorrentState(infohash=unhexlify(b'0003'), seeders=1, leechers=2, self_checked=1, has_data=1,
+                         last_check=now - 1000)
+        mds.TorrentState(infohash=unhexlify(b'0004'), seeders=1, leechers=2, self_checked=0, has_data=1,
+                         last_check=now + 1000)
+        mds.TorrentState(infohash=unhexlify(b'0005'), seeders=1, leechers=2, self_checked=1, has_data=1,
+                         last_check=now + 1000)
```

Comment:
It is not clear which parameters are important for the test and which are not.
I assume there are two important parameters:
* self_checked
* last_check

Therefore I suggest changing this code part to the following:
```suggestion
    now = int(time.time())
    in_the_past = now - 1000
    in_the_future = now + 1000

    mds = MetadataStore(mds_path, channels_dir, trustchain_keypair, check_tables=False)

    @db_session
    def _add_torrent_state(self_checked=1, last_check=0):
        mds.TorrentState(infohash=random.randbytes(20), seeders=0, leechers=0, self_checked=self_checked, has_data=0,
                         last_check=last_check)

    _add_torrent_state(self_checked=0, last_check=0)
    _add_torrent_state(self_checked=0, last_check=in_the_past)
    _add_torrent_state(self_checked=1, last_check=in_the_past)
    _add_torrent_state(self_checked=0, last_check=in_the_future)
    _add_torrent_state(self_checked=1, last_check=in_the_future)
```

Reply:
Agreed, changed the test this way (with minor differences)